yt_dlp/extractor/opencast.py

```diff
@@ -87,7 +92,7 @@ def _parse_mediapackage(self, video):
                 audio_info = track.get('audio')
                 if audio_info is not None:
                     track_obj.update({
-                        'abr': int_or_none(audio_info.get('bitrate'), 1000),
+                        'abr': int_or_none(audio_info.get('bitrate')),
```

Comment:
yo revert this
sorry i didn't realize this is actually scale and not default parameter for int_or_none 😭